recipe/meta.yaml

```
@@ -87,10 +87,15 @@ requirements:
     - aws-sdk-cpp
 
 test:
+  files:
+    - test_netcdf.py
+  requires:
+    - netcdf4
   commands:
     - test ! -f ${PREFIX}/lib/libnetcdf.a          # [not win]
     - test -f ${PREFIX}/lib/libnetcdf${SHLIB_EXT}  # [not win]
     - nc-config --all  # [not win]
+    - python -c "import test_netcdf; test_netcdf.test_netcdf"
```

Shouldn't this be the following?
```suggestion
    - python -c "import test_netcdf; test_netcdf.test_netcdf()"
```
Isn't it just doing nothing with the function pointer if the parentheses are missing?